<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>478</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>802</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>AcadEref</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lab_pic">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_name">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="text">
         <string>北京首赫防务科技有限公司</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_display_debug">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::indicator:checked {background-color:red;}
QRadioButton::indicator:unchecked {background-color:white;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Alt+D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_time">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>12:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lab_line">
      <property name="minimumSize">
       <size>
        <width>780</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icon.qrc">:/Icon/doubleline.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>770</width>
        <height>280</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbut_Network">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:black; color:white; border-radius:10px; border:2px groove gray; border-style:outset;text-align:bottom; font-size:25px;}
                           QPushButton:hover{background-color:white; color:black;}
                           QPushButton:pressed{background-color:rgb(85, 170, 255); border-style:inset; }</string>
        </property>
        <property name="text">
         <string> 网 络</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/Icon/network_tm.png</normaloff>:/Icon/network_tm.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbut_Location">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:black; color:white; border-radius:10px; border:2px groove gray; border-style:outset;text-align:bottom; font-size:25px;}
                           QPushButton:hover{background-color:white; color:black;}
                           QPushButton:pressed{background-color:rgb(85, 170, 255); border-style:inset; }</string>
        </property>
        <property name="text">
         <string> 位 置</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/Icon/geolocation3_tm.png</normaloff>:/Icon/geolocation3_tm.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbut_Compass">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:black; color:white; border-radius:10px; border:2px groove gray; border-style:outset;text-align:bottom; font-size:25px;}
                           QPushButton:hover{background-color:white; color:black;}
                           QPushButton:pressed{background-color:rgb(85, 170, 255); border-style:inset; }</string>
        </property>
        <property name="text">
         <string>方 位</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/Icon/compass1.png</normaloff>:/Icon/compass1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbut_Detectors">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:black; color:white; border-radius:10px; border:2px groove gray; border-style:outset;text-align:bottom; font-size:25px;}
                           QPushButton:hover{background-color:white; color:black;}
                           QPushButton:pressed{background-color:rgb(85, 170, 255); border-style:inset; }</string>
        </property>
        <property name="text">
         <string> 侦 测</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/Icon/detector4_tm.png</normaloff>:/Icon/detector4_tm.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbut_Shields">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color:black; 
color:white; border-radius:10px; 
border:2px groove gray; border-style:outset;text-align:bottom; font-size:25px;}
                           QPushButton:hover{background-color:white; color:black;}
                           QPushButton:pressed{background-color:rgb(85, 170, 255); border-style:inset; }</string>
        </property>
        <property name="text">
         <string> 防 御</string>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/Icon/defence3_tm.png</normaloff>:/Icon/defence3_tm.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
